core/communication_manager.py

```diff
@@ -79,7 +79,7 @@ def inform(self, event_type, payload):
         if self.optimizer:
             # We have an active session.
             event_type, payload = self.optimizer.ask(event_type, payload)
-            callback = callback_handler_no_default(
+            callback = callback_handler(
```

Comment:
@kiddyboots216: Next time someone makes a string of commits to a branch you're working on, don't "solve merge conflicts" because there's nothing to solve. There are local changes you need to discard like this one.

Fixing.